packages/compass-connections/src/components/form-help/form-help.tsx

```diff
@@ -19,8 +19,7 @@ const { track } = createLoggerAndTelemetry('COMPASS-CONNECT-UI');
 const formHelpContainerStyles = css({
   position: 'relative',
   margin: 0,
-  minWidth: 250,
-  maxWidth: 400,
+  width: 350,
```

Comment:
`spacing[5] * 10`?

Should we use `spacing[x]` everywhere at least by default? that would make the size of elements proportionate with the spacings between them and in theory should look better.

Btw the window size like this still can't be `1280x800` (mac 13''). I played around a bit and looks like these widths + gaps would fit 1280 and still look ok:
- sidebar: `spacing[4] * 10 + spacing[2]` (248)
- form: `spacing[6] * 10` (640)
- help: `spacing[5] * 10` (320)

Should we just have a different PR for this that would include also the resized window and ping @Sgrinfy on that?